README.md

```diff
@@ -62,19 +55,18 @@ All system contracts will be flattened and output into `${workspace}/contracts/f
 
 1. Edit `init_holders.js` file to alloc the initial BNB holder.
 2. Edit `validators.js` file to alloc the initial validator set.
-3. Edit `generate-validatorset.js` file to change `fromChainId` and `toChainId`,
-4. Edit `generate-tokenhub.js` file to change `refundRelayReward`, `minimumRelayFee` and `maxGasForCallingBEP20`.
-5. Edit `generate-tendermintlightclient.js` file to change `chainID` and `initConsensusStateBytes`.
-6. run ` node generate-genesis.js` will generate genesis.json
+3. Run `bash scripts/generate-*.sh` to change system contracts setting.
+4. Run `node scripts/generate-genesis.js` will generate genesis.json
 
 ## How to generate mainnet/testnet/QA genesis file
 
 ```shell 
-npm run generate-mainnet
-npm run generate-testnet
-npm run generate-QA
+bash scripts/generate.sh mainnet
+bash scripts/generate.sh testnet
```

Comment:
currently it will generate `CrossChain.sol-e` and other `-e` files, can we aovid that?